lighthouse-core/scripts/generate-bundlephobia-database.js

```diff
@@ -0,0 +1,154 @@
+/**
+ * @license Copyright 2020 The Lighthouse Authors. All Rights Reserved.
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License. You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
+ */
+
+'use strict';
+
+/* eslint-disable no-console */
+
+// TODO(saavan) These type definitions are incorrect and need to be fixed.
```

Comment:
are these the one's we're committing in this same PR? maybe worth typing as `unknown` or something in the meantime to signify what's broken if it's too much work to take on right now?

if I had to takeover this today I wouldn't know where to start in fixing that typedef :)